modules/openxr/action_map/openxr_interaction_profile.cpp

```diff
@@ -81,6 +87,76 @@ String OpenXRIPBinding::get_binding_path() const {
 	return binding_path;
 }
 
+int OpenXRIPBinding::get_binding_modifier_count() const {
+	return binding_modifiers.size();
+}
+
+Ref<OpenXRBindingModifier> OpenXRIPBinding::get_binding_modifier(int p_index) const {
+	ERR_FAIL_INDEX_V(p_index, binding_modifiers.size(), nullptr);
+
+	return binding_modifiers[p_index];
+}
+
+void OpenXRIPBinding::clear_binding_modifiers() {
+	// Binding modifiers held within our interaction profile set should be released and destroyed but just in case they are still used some where else.
+	if (binding_modifiers.size() == 0) {
+		return;
+	}
+
+	for (int i = 0; i < binding_modifiers.size(); i++) {
+		Ref<OpenXRBindingModifier> binding_modifier = binding_modifiers[i];
+		binding_modifier->ip_binding = nullptr;
+	}
+	binding_modifiers.clear();
+	emit_changed();
+}
+
+void OpenXRIPBinding::set_binding_modifiers(const Array &p_binding_modifiers) {
+	clear_binding_modifiers();
+
+	// Any binding modifier not retained in p_binding_modifiers should be freed automatically, those held within our Array will have be relinked to our interaction profile.
+	for (int i = 0; i < p_binding_modifiers.size(); i++) {
+		// Add them anew so we verify our binding modifier pointer.
+		add_binding_modifier(p_binding_modifiers[i]);
+	}
+}
+
+Array OpenXRIPBinding::get_binding_modifiers() const {
+	Array ret;
+	for (const Ref<OpenXRBindingModifier> &binding_modifier : binding_modifiers) {
+		ret.push_back(binding_modifier);
+	}
+	return ret;
+}
```

Comment:
Because we're converting from a `Vector<Ref<OpenXRBindingModifier>>` to an `Array`. I don't think there is an implicit conversion or was that added at some time?